### DISCUSSION TITLE
Version 3 migration question on assets

### DISCUSSION THREAD

**Discussion by brockfanning** [uswds/uswds#6934] [Q&A] [answered]
Hello, in version 2 I believe there was an icon at the path: /img/angle-arrow-down-primary.svg. In version 3, has this icon been removed? I didn't see anything about it in the migration docs, but I'm not seeing it in our compiled "assets/img" folder. There are other images too but I suspect any answer to this one will resolve the other ones too. Thank you for any advice!

**Comment by mejiaj** [comment, marked as answer]
Hey @brockfanning, this icon was moved & renamed in `release-3.0.0-beta.4`. Currently, it's replaced by `expand_more` and `expand_less`.

All legacy icons are currently in `packages/usa-icon/src/img/usa-icons-bg`.

**Comment by ryparker** [comment]
I've moved this to Q&A and marked the explanation of the renamed icons as the answer. The original question and replies are all here, so please continue here if anything is still unclear.
